manifest/ocischema/manifest_test.go

```diff
@@ -87,6 +90,9 @@ func TestManifest(t *testing.T) {
 	if !reflect.DeepEqual(&unmarshalled, deserialized) {
 		t.Fatalf("manifests are different after unmarshaling: %v != %v", unmarshalled, *deserialized)
 	}
+	if deserialized.Annotations["hot"] != "potato" {
```

Comment:
@stevvooe said "This is a policy question. Policy should be applied regardless of the type." I couldn't find any documentation on URL filtering. Maybe I was looking in the wrong place?  My earlier attempt to map policy relied solely on defacto policy derived from the code and a presumption for how to map that to OCI Schema. 

I agree with you and @stevvooe that such policies should apply regardless of type. So my question is, is this an official policy or defacto and how should the policy be mapped to the OCI Schema? Was my earlier code correct? Should I put it back in? Is there a place where this policy is documented?